soh/soh/resource/importer/AudioSampleFactory.cpp

```diff
@@ -66,6 +73,62 @@ void LUS::AudioSampleFactoryV0::ParseFileBinary(std::shared_ptr<BinaryReader> re
     audioSample->book.book = audioSample->bookData.data();
     audioSample->sample.book = &audioSample->book;
 }
+
+void LUS::AudioSampleFactoryV1::ParseFileBinary(std::shared_ptr<BinaryReader> reader,
+                                                 std::shared_ptr<IResource> resource)
+{
+    std::shared_ptr<AudioSample> audioSample = std::static_pointer_cast<AudioSample>(resource);
+    ResourceVersionFactory::ParseFileBinary(reader, audioSample);
+
+    reader->Seek(52, SeekOffsetType::Start);
```

Comment:
Just a note for this; I assume the offset is 52 here because of the otr header added to the file (on top of the wave tag location offset)? If so I think either a comment or something explaining the reasoning for this for someone who would be looking at this section later on would be helpful (wasnt really clear to me until I read the otr-iser stuff)

Reply:
if you `#define` it and give it a good name that'd be perfect. named values are even better than explained magic numbers.